### ISSUE TITLE
Converting between equivalent calendars failed 

### ISSUE THREAD

**Issue by jvegreg** [SciTools/cf-units#80]
Hi

CF_units is failing when  trying to convert from a time reference using a gregorian calendar to a time reference using a standard calendar:

```
  File "/anaconda/envs/esmval/lib/python2.7/site-packages/cf_units/__init__.py", line 2046, in convert
    (self, other))
ValueError: Unable to convert from 'Unit('days since 1850-1-1', calendar='gregorian')' to 'Unit('days since 1950-01-01 00:00:00', calendar='standard')'.
```
Those calendar are just different aliases for the same calendar, by CF-conventions: 

http://cfconventions.org/Data/cf-conventions/cf-conventions-1.7/build/ch04s04.html

It will be nice if this equivalencies are handled directly by cf_units not only for those two, but also for all the other cases (noleap and 365_day, for example)

Thanks,
Javi

**Comment by bjlittle** [comment]
Hi @jvegasbsc 

Yes, agreed. It would be useful for the calendar aliases to work as expected.

I'll take a look for you ...

**Comment by pelson** [comment]
Hi @jvegasbsc. We've just added this functionality into cf_units, and it will be available in v2.0.

Cheers,

Phil
